README.md

```diff
@@ -3,7 +3,6 @@
 [![Affiliated with NumFOCUS](https://img.shields.io/badge/NumFOCUS-affiliated%20project-orange.svg?style=flat&colorA=E1523D&colorB=007D8A)](https://numfocus.org/sponsored-projects/affiliated-projects)
 [![PyPI](https://img.shields.io/pypi/v/signac.svg)](https://pypi.org/project/signac/)
 [![conda-forge](https://img.shields.io/conda/vn/conda-forge/signac.svg?style=flat)](https://anaconda.org/conda-forge/signac)
-![CircleCI](https://img.shields.io/circleci/project/github/glotzerlab/signac/master.svg)

```

Comment:
We should put the GHA badge here instead

Reply:
Done.